packages/pulpcore-obsolete-packages/pulpcore-obsolete-packages.spec

```diff
@@ -0,0 +1,28 @@
+Name: pulpcore-obsolete-packages
+Version: 1.0
+Release: 1%{?dist}
+License: MIT
+Summary: A package to obsolete retired packages
+URL: https://github.com/theforeman/pulpcore-packaging
+BuildArch: noarch
+
+Obsoletes:      python3-django-currentuser
+%if 0%{?rhel} == 8
+Obsoletes:      python39-django-currentuser
```

Comment:
Can you update this to have an upper version bound, so if we ever want to introduce the package back, we can?

(Same for the el9 one)